stories/Templates/Authors/Authors.scss

```diff
@@ -149,7 +149,7 @@
   }
 }
 
-[dir='rtl'] {
+[dir="rtl"] {
```

Comment:
@Ashish-srij code should be linted as it should have single quotes instead of double quotes